### PR TITLE
feat(presets): add more JS testing packages

### PR DESCRIPTION
**Pull request by acifani**
## Changes:

- Created `testing-library` package
- Add `testing-library` package and `enzyme` to `jsUnitTest` package

## Context:

These are very popular JS/React test libraries.

Repos:
- https://github.com/testing-library
- https://github.com/enzymejs/enzyme

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/presets/internal/packages.ts
Comment (viceice) [inline lib/config/presets/internal/packages.ts:54]: Why a new preset? Simply add `matchPackagePrefixes` to the other preset.

**Review by viceice — CHANGES_REQUESTED** [review]

File: lib/config/presets/internal/packages.ts
Comment (acifani) [reply lib/config/presets/internal/packages.ts:54]: I thought it would be useful to have them as a standalone package as well, but yeah, I can change that no problem

**Comment by HonkingGoose** [comment]
@rarkins This PR is stuck in auto-merge hell, because it still needs an approving review from a reviewer with write access.

**Review by viceice — APPROVED** [review]

**Pull request merged**
